ajax/libs/overpass/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "overpass",
   "version": "3.0.2",
-  "filename": "overpass-webfont/overpass.css",
+  "filename": "overpass-webfont/overpass.min.css",
```

Comment:
@dakshshah96 I think you can try to fix the conflict. You can take a look at https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/